documentation-website/cypress/e2e/sponsors.cy.ts

```diff
@@ -11,7 +11,8 @@ describe('contributing/sponsors', () => {
     cy.url().should('eq', BASE_URL + page.sponsors,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Sponsors | Contributing | @idrinth/api-bench',);
+    cy.title().should('eq', 'Sponsors | Contributing | '+
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted